tools/runner/index.html

```diff
@@ -91,6 +91,12 @@
     </div>
   </div>
 
+  <div class="warning">
```

Comment:
Thanks for working on this! I had a look at https://getbootstrap.com/docs/3.3/components/#panels and the bootstrap.css in this repo, and think this structure might result in a more warning-looking styling:

```
  <div class="alert alert-warning">
      This runner does not fully support all test types and has a number of
      <a href="https://github.com/web-platform-tests/wpt/labels/runner">known issues</a>.
      <code>./wpt run</code> is a more well-supported runner, see the documentation on
      <a href="https://web-platform-tests.org/running-tests/">running tests</a>.
  </div>
```

WDYT?

If you like that, can you put it before big controls box, first thing in `<div class="container">`?

Reply:
Hi @foolip thanks for the review and suggestions :smile: it makes sense to me , let me make changes and share the screenshot for the same :100:

Reply:
Its looking like this after making the suggested changes.

![image](https://user-images.githubusercontent.com/43577303/47782652-c184c600-dd26-11e8-8c95-5199e6cec9c8.png)

Reply:
That looks great, thanks!